src/vm/exceptionhandling.cpp

```diff
@@ -4945,74 +4945,77 @@ bool IsDivByZeroAnIntegerOverflow(PCONTEXT pContext)
 
 VOID PALAPI HandleHardwareException(PAL_SEHException* ex)
 {
-    if (ex->ExceptionRecord.ExceptionCode != STATUS_BREAKPOINT && ex->ExceptionRecord.ExceptionCode != STATUS_SINGLE_STEP)
+    if (g_fEEStarted)
```

Comment:
My preference is to use early return in cases like this one.  http://llvm.org/docs/CodingStandards.html#use-early-exits-and-continue-to-simplify-code is a good description on why.